doc/WebSite/Validating-Data-Transfer-Objects.md

```diff
@@ -111,6 +111,60 @@ IValidatableObject interface. You can also implement it to perform
 additional custom validations. If you implement both interfaces, both of
 them will be called.
 
+### Fluent Validation
+
+In order to use [FluentValidation](https://github.com/JeremySkinner/FluentValidation), you need to install [Abp.FluentValidation](https://www.nuget.org/packages/Abp.FluentValidation) package first.
+
+```
+Install-Package Abp.FluentValidation
+```
+
+Then, You should set the dependency to AbpFluentValidationModule from your module. Example:
+
+```
+[DependsOn(typeof(AbpFluentValidationModule))]
+public class MyProjectAppModule : AbpModule
+{
+	
+}
+```
+
+After all, you can define your [FluentValidation](https://github.com/JeremySkinner/FluentValidation) validators to validate matching input classes.
+
+As an example, if we have an input class and a Controller which uses this class as it's input parameter;
+
+```
+public class MyCustomArgument1
+{
+	public int Value { get; set; }
+}
+
+public class MyTestController : AbpController {
+
+	public JsonResult GetJsonValue([FromQuery] MyCustomArgument1 arg1)
+	{
+		return Json(new MyCustomArgument1
+		{
+			Value = arg1.Value
+		});
+	}
+}
+```
+
+If we want to limit the value of MyCustomArgument1's Value field between 1 and 99, we can define a validator like the one below;
+
+```
+public class MyCustomArgument1Validator : AbstractValidator<MyCustomArgument1>
+{
+	public ValidationTestArgument1Validator()
```

Comment:
Constructor name should be MyCustomArgument1Validator

Reply:
Thanks @ogix , done.